docs/en/Architecture-DataFlow.md

```diff
@@ -36,7 +36,7 @@ Hadoop MapReduce).  There are multiple benefits to have this layer in the stack:
   layer for any number of varied data sources.
 
 At a high level, Alluxio can be divided into three components: the master,
-workers, and clients.  Alluxio consists a single primary master and multiple
+workers, and clients.  Alluxio consists a single leader master and multiple
```

Comment:
This part is kind of confusing, since we could have standbys as well.

Reply:
updated to mention the standby masters here